src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const basePrice = 40;
+  const shortTerm = 3;
+  const longTerm = 7;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < shortTerm) {
+    return days * basePrice;
+  } else if (days < longTerm) {
+    return days * basePrice - smallDiscount;
+  } else {
```

Comment:
Ensure that the condition for applying the big discount aligns with the task requirements. If the task specifies a different threshold or condition, update this logic.